commerce_coordinator/apps/rollout/utils.py

```diff
@@ -0,0 +1,17 @@
+""" Rollout Utility Functions"""
+
+import re
+
+
+def is_legacy_order(order_number: str) -> bool:
+    """ Determine if an order is a Legacy Ecommerce order """
+    if not order_number:
+        return False
```

Comment:
IMO, this check should be added to the `is_legacy_order`'s caller code block.

Reply:
Utility functions usually assume a caller can be fallible.